src/components/EasyDropdown/story.js

```diff
@@ -96,6 +96,26 @@ storiesOf('Galaxies/EasyDropdown', module)
           </EasyDropdown>
           <br />
           <br />
+          <EasyDropdown
+            menuItems={[{ label: 'Click me!', onClick: () => {} }]}
+            defaultIsOpen={false}
+          >
+            Button with addon
+            <Button.Addon>→</Button.Addon>
+          </EasyDropdown>
+          <br />
+          <br />
+          <EasyDropdown
+            menuItems={[{ label: 'Click me!', onClick: () => {} }]}
+            defaultIsOpen={false}
+            style={{ width: '100%' }}
+            toggleProps={{ style: { width: '100%' } }}
```

Comment:
I don't love the need to declare `width: 100%` twice, but I didn't have any better ideas, considering the `EasyDropdown` has both it's own wrapping div _and_ an element for the toggle button.